xtra-libp2p/src/dialer.rs

```diff
@@ -7,6 +7,7 @@ use async_trait::async_trait;
 use libp2p_core::Multiaddr;
```

Comment:
I don't think it's possible to find out which commit you were referring to using github UI - please quote the commit or hash in multi-commit PRs, otherwise it's hard to respond!

Reply:
Oops, I might the wrapping in...

...between commit title and message.